Services/CO.CDP.OrganisationInformation.Persistence/DatabaseTenantRepository.cs

```diff
@@ -48,7 +48,8 @@ public void Save(Tenant tenant)
                         Id = o.Guid,
                         Name = o.Name,
                         Roles = o.Roles,
-                        Scopes = o.OrganisationPersons.Single(op => op.PersonId == p.Id).Scopes
+                        // ReSharper disable once NullCoalescingConditionIsAlwaysNotNullAccordingToAPIContract
```

Comment:
Ignored a rider inspection as it is a false positive. The test proves null value is possible here.